code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -409,6 +409,7 @@ MATCHBOXES ARE ALSO IN FANCY.DM
 	item_state = "cigar"
 	smoketime = 1500
 	chem_volume = 20
+	species_fit = list(VOX_SHAPED)
```

Comment:
Will this not stop humans smoking them? I believe you have to use list("include", VOX_SHAPED), or maybe there's a define like INCLUDE.